app/scripts/models/reliers/relier.js

```diff
@@ -21,7 +21,15 @@ define([
   Constants) {
   'use strict';
 
-  var RELIER_FIELDS_IN_RESUME_TOKEN = ['campaign', 'entrypoint'];
+  var RELIER_FIELDS_IN_RESUME_TOKEN = [
+    'utmTerm',
+    'utmSource',
+    'utmMedium',
+    'utmContent',
+    'utmCampaign',
+    'campaign',
```

Comment:
this is going to be going away soon via https://github.com/mozilla/fxa-content-server/issues/2920